src/core_plugins/kibana/public/visualize/listing/visualize_listing.html

```diff
@@ -39,7 +39,7 @@
           ng-click="listingController.deleteSelectedItems()"
           tooltip="Delete selected visualizations"
         >
-          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-trash"></span>
+          <react-component name="TrashIconComponent"></react-component>
```

Comment:
Was a conscious choice made here about using `<react-component>` over the [reactDirective Service](https://github.com/ngReact/ngReact#the-reactdirective-service)? The latter seems to allow more control over the way bindings are converted to props.